pages/infrastructure_providers-validators/_meta.json

```diff
@@ -2,7 +2,7 @@
   "hardware_requirements": "Hardware Requirements",
   "required_node_configs": "Required Node Configs",
   "how_to_set_up_full_node": "Set Up a Full Node",
-  "running_full_node": "Run a Full Node",
+  "running_full_node": "Optimize Your Full Node",
```

Comment:
if/when you do this can you add a redirect to `next.config.js`? some paths are indexed in google/etc and used in other places, we should try to keep the original paths